src/main/java/com/xpanxion/java/springboot/da1/demo/model/student1/WorkoutHistory1.java

```diff
@@ -0,0 +1,63 @@
+package com.xpanxion.java.springboot.da1.demo.model.student1;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
+@Table(name = "workoutHistory1")
+public class WorkoutHistory1 {
+
+    @Id
+    @SequenceGenerator(
+            name = "student1_workoutHistorySequence",
+            sequenceName = "student1_workoutHistorySequence",
+            allocationSize = 1
+    )
+    @GeneratedValue(
+            strategy = GenerationType.SEQUENCE,
+            generator = "student1_workoutHistorySequence"
+    )
+    private int id;
+    private int memberId;
```

Comment:
> I wasn't sure how else to retrieve the data from the data base without a query. Is there no need for this approach?

Instead of using an `int memberId` here we can use a `Member1 member`, and then we can use the magic of JPA:

`timestampsRepository1.findByMember0(member)`  

This gets all the timestamps for the member and there's no reason for the `JdbcTemplate`.

Reply:
`timestampsRepository1.findByMember1(member)`

Reply:
Oh wow, I didn't know that. Thanks, making changes now